src/main.js

```diff
@@ -1,16 +1,16 @@
 import "regenerator-runtime/runtime";
 
-import * as nearlib from "nearlib"
+import * as nearAPIJs from "near-api-js"
```

Comment:
Note that on docs.nearprotocol.com, it [still says](https://github.com/nearprotocol/docs/pull/295/files) `nearlib` here. I like the idea of calling it something else, since the library is no longer called `nearlib`, but I'm not convinced that `nearAPIJs` is the best thing to call it. I'd rather just `nearapi` or possibly import the individual methods needed (`import { connect, WalletAccount } from 'near-api-js'`). But if this conversation isn't super quick, then let's skip having it for now and keep calling it `nearlib` in the code, as the docs do.